MotMystere.cpp

```diff
@@ -2,19 +2,53 @@
 //
 
 #include <iostream>
+#include <string>
+#include <ctime>
+#include <cstdlib>
+
+using namespace std;
 
 int main()
 {
-    std::cout << "Hello World!\n";
+   
+    
+    //1 : On demande de saisir un mot
```

Comment:
Generally speaking, comments are often not used properly. 

Comments can be useful is specific cases, like documentation or specifying a complex algorithm.
But comments are not compiled, so they can become obsolete when you change the code if you don't update the code as well. This is why it is preferred to have your code be 'explicit in intent'.

For example, instead of coding a step and defining a comment like 'On demande de saisir un mot', it would be better to declare a function called 'askUserForAWord()' and put all the code for this step in this function.

This way, when someone reads the function, they will see 'askUserForAWord()' and immediatly understand what is going on

Reply:
ok